### ISSUE TITLE
UnreservedConcurrentExecution below its minimum value of [10]

### ISSUE THREAD

**Issue by Sgnarf** [aws-samples/amazon-rekognition-serverless-large-scale-image-and-video-processing#14]
I've seen that this issue has been opened before: https://github.com/aws-samples/amazon-rekognition-serverless-large-scale-image-and-video-processing/issues/6

Is it possible to change the code so that it doesn't require to open a ticket to AWS? 

Seems like the issue comes from this file: RekognitionPipelineStack.template, but if I change it manually it gets overwritten when I try to deploy

**Comment by hammzagee** [comment]
I don't think you can manually change the limit in the code, as you have mentioned it gets overwritten.

you gotta open a ticket with AWS support.
